THIRD-PARTY.md

```diff
@@ -0,0 +1,65 @@
+See LICENSE for the main open source license of original code written for sphinx_press_theme.
+
+The remainder of this file reproduces the open source licensing details of other projects that have been imported, incorporated into, or derived into parts of sphinx_press_theme.
+
+In no particular order:
+
+----
+
+sphinx-themes, courtesy of WAKAYAMA Shirou
```

Comment:
This project uses anything from sphinx-themes ?
I guess it should the original sphinx project...